TDlist.test.js

```diff
@@ -0,0 +1,42 @@
+import displayList from './__mocks__/displayList';
+import TDlist from './src/TDlist';
+
```

Comment:
- [ ] On your readme, please, kindly add the "Run tests" section, according to the [Microverse Readme Template](https://github.com/microverseinc/readme-template).  

## Explanation

On every project that requires tests, the "Run tests" section of the Microverse Readme Template is required.

That way, your hard work will be well documented! 💪